mangaki/mangaki/utils/nmf.py

```diff
@@ -0,0 +1,54 @@
+from mangaki.utils.chrono import Chrono
+from sklearn.decomposition import NMF
+from scipy.sparse import lil_matrix
+import numpy as np
+import pandas
+
+
+class MangakiNMF(object):
+    M = None
+    W = None
+    H = None
+    def __init__(self, NB_COMPONENTS=10):
+        self.NB_COMPONENTS = NB_COMPONENTS
+        self.chrono = Chrono(True)
+        self.works = pandas.read_csv('data/works.csv', header=None).as_matrix()[:, 1]
```

Comment:
Can we use `settings.BASE_DIR` to find the CSV here ? It'd be great to not have an implicit requirement on the directory you must be in to run `manage.py` commands.